cql3/statements/alter_table_statement.hh

```diff
@@ -43,12 +45,22 @@ private:
     const std::vector<column_change> _column_changes;
     const std::optional<cf_prop_defs> _properties;
     const renames_type _renames;
+
+    const std::optional<api::timestamp_type> _drop_timestamp;
+    const std::unique_ptr<attributes::raw> _attrs;
```

Comment:
Should be split to alter_table_statement and raw::alter_table_statement (better in a separate patch), see create_table_statement. I understand the reluctance to do it but it's better in the long run that we can distinguish between the prepared and unprepared parts of a statement.

Reply:
done